ansible/roles/k3s/tasks/main.yml

```
@@ -0,0 +1,43 @@
+---
+
+- name: Download k3s install script
+  ansible.builtin.get_url:
+    url: https://get.k3s.io/
+    timeout: 120
+    dest: /usr/bin/k3s-install.sh
+    owner: root
+    group: root
+    mode: "0755"
+
+- name: Install k3s
+  ansible.builtin.shell:
+    cmd: /usr/bin/k3s-install.sh
+  environment:
+    INSTALL_K3S_VERSION: "v1.31.0+k3s1"
+    INSTALL_K3S_EXEC: "{{ item }}"
+    INSTALL_K3S_SKIP_START: "true"
+    INSTALL_K3S_SKIP_ENABLE: "true"
+    INSTALL_K3S_BIN_DIR: "/usr/bin"
+  changed_when: true
+  with_items:
+    - server
+    - agent
+
+- name: Installing helm
+  unarchive:
+    src: https://get.helm.sh/helm-v3.11.0-linux-amd64.tar.gz
+    dest: /usr/bin
+    extra_opts: "--strip-components=1"
+    owner: root
+    group: root
+    mode: 0755
+    remote_src: true
+
+- name: k9s install
```

```suggestion
- name: Install k9s
```
Just for consistency